<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardEvaluator</class>
 <widget class="QWidget" name="CardEvaluator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>580</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>card evaluator</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>kleur</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>299</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="color">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>maakt niet uit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>harten</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>klaveren</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>schoppen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ruit</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Selecteer een kleur uit de lijst of type een variabele in. Een variabele begint altijd met een kleine letter. </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>481</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>waarde</string>
   </property>
   <widget class="QPushButton" name="addValue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>add value</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>operator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="prefix">
         <item>
          <property name="text">
           <string>=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>!=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;=</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&gt;=</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Waarde</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="value">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>J</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Q</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>A</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>postfix</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="postfix_plus">
           <property name="text">
            <string>+</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="postfix_minus">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="postfix">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Selecteer een (on)gelijkheid uit de lijst en de gewenste waarde van de kaart. Als de waarde een variabele is, kan er gekozen worden voor een postfix. Een variabele begint altijd met een kleine letter.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QScrollArea" name="cards">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>361</width>
     <height>351</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>349</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="newCard">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>add card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCard">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete card</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
